packages/ag-charts-website/src/utils/pages.ts

```diff
@@ -169,8 +164,7 @@ export const getBoilerPlateUrl = ({
  * Get dev files for local development
  */
 export function getDevFiles(): DevFileRoute[] {
-    const files = Object.keys(DEV_FILE_PATH_MAP).map((filePath) => {
-        const sourceFilePath = DEV_FILE_PATH_MAP[filePath];
+    const files = Object.entries(DEV_FILE_PATH_MAP).map(([filePath, sourceFilePath]) => {
```

Comment:
👍🏼 I'm sure I've not used `Object.entries` in other places where it would've been useful too